pandas/core/series.py

```diff
@@ -1809,7 +1805,7 @@ def unique(self):
         result = super().unique()
         return result
 
-    def drop_duplicates(self, keep="first", inplace=False) -> "Series":
+    def drop_duplicates(self, keep="first", inplace=False) -> Optional["Series"]:
```

Comment:
are we actually testing the return value of inplace ops for drop_duplicates / duplciated? IIRC these are actually returning an object (at least they *were*).

Reply:
> are we actually testing the return value of inplace ops for drop_duplicates / duplciated? 

Yes, editing this to return self instead of None breaks a test in tests.frame.test_api